libs/service-portal/regulations-admin/src/state/useDraftingState.ts

```diff
@@ -474,7 +474,7 @@ export const useDraftingState = (
                 text: apx.text.value,
               })),
               comments: draft.comments.value,
-              ministryId: draft.ministry.value
+              ministry: draft.ministry.value
                 ? state.ministries.find((m) => m.name === draft.ministry.value)
                     ?.name
                 : undefined,
```

Comment:
```suggestion
              ministry: draft.ministry.value,
```

BTW, This code was _supposed_ to temporarily map to `?.slug` in line 479, but I accidentally mapped back to `.name` 🙈 

This mapping is now wholly unneccessary